Client/Frontend/Widgets/OneLineCellAndFooter.swift

```diff
@@ -20,7 +20,7 @@ struct OneLineTableViewCellViewModel {
     let accessoryType: UITableViewCell.AccessoryType
 }
 
-class OneLineTableViewCell: UITableViewCell, NotificationThemeable, ReusableCell {
+class OneLineTableViewCell: UITableViewCell, NotificationThemeable, ReusableCell, ThemeApplicable {
```

Comment:
This work will be affected by https://github.com/mozilla-mobile/firefox-ios/pull/12245, and if possible we should remove NotificationThemeable here (or create another task for it if its too messy to remove?)